packages/checkout/sdk/src/blockExplorer/blockExplorer.ts

```diff
@@ -0,0 +1,16 @@
+import { BLOCKSCOUT_CHAIN_URL_MAP } from '../env';
+import { ChainId } from '../types';
+
+export class BlockExplorerService {
```

Comment:
Adding this service since the BLOCKSCOUT_CHAIN_URL_MAP is not exported from sdk.
Please let me know if we better do it a different way

Reply:
Hmm yep this makes sense. Not sure if we want to expose the Blockscout instance but we could expose the Blockscout instance URL if you pass in the chainId. Would be better then reconstructing it maybe?